services/web-server/src/resolvers/Notify.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  Query: {
+    listDenylistAddresses(parent, { connection, filter }, { loaders }) {
+      return loaders.notify.load({ connection, filter });
```

Comment:
This is happening because your available loader is called `denylist` not `notify`.  `loaders.denyList.load` should work.

Reply:
So after renaming the loader i should now call `loaders.listDenylistAddresses.load` right?

Reply:
Yep.